docs/rpm_packages.md

```diff
@@ -0,0 +1,15 @@
+# MicroShift RPM Packages
+
+## Development Packages
+> Connect to the Red Hat Virtual Private Network (VPN) for accessing the development packages.
+
+MicroShift RPM development packages are intended to be used internally at Red Hat to facilitate the development and QE processes, **never** to be distributed to customers. 
+
+The packages are built every <what's the cadence?>. Review the [MicroShift Brew Package](https://brewweb.engineering.redhat.com/brew/packageinfo?packageID=82827) page for more information about the product builds and tags. 
+
+The packages can be downloaded from the [MicroShift Brew Download](http://download.eng.bos.redhat.com/brewroot/vol/rhel-8/packages/microshift) page. The builds are rotated every <what's the cadence?> days.
```

Comment:
@sosiouxme / @thiagoalessio 
Could you comment on package build and rotation cadence?

Reply:
I think they're nightly, right?